### ISSUE TITLE
Auto prune is not working, i think?

### ISSUE THREAD

**Issue by JackDinn** [borgbase/vorta#1574]
Vorta 0.8.10
borg 1.2.3
Kernel: 6.0.19-3-MANJARO arch: x86_64 bits: 64
Desktop: KDE Plasma v: 5.26.5 Distro: Manjaro Linux

Iv just noticed that i dont seem to be able to get vorta to prune any more.
I look at the logs:-

`2023-01-31 20:35:51,839 - vorta.borg.borg_job - INFO - Running command /usr/bin/borg prune --list --info --log-json --keep-hourly 0 --keep-daily 10 --keep-weekly 0 --keep-monthly 6 --keep-yearly 0 -a sh:greg-inspiron5767-* --keep-within 3d /mnt/TRUENAS/manjaro-home/borg`

So Vorta is sending this borg command but the `-a sh:greg-inspiron5767-*` is causing a problem.

If i run this on bash
`borg prune --dry-run --list --info --keep-hourly 0 --keep-daily 10 --keep-weekly 0 --keep-monthly 6 --keep-yearly 0 -a sh:greg-inspiron5767-* --keep-within 3d /mnt/TRUENAS/manjaro-home/borg`

I get nothing, not even an error.

If i run the same command in fish (my main shell) i get a little more
```
❱borg prune --dry-run --list --info --log-json --keep-hourly 0 --keep-daily 10 --keep-weekly 0 --keep-monthly 6 --keep-yearly 0 -a sh:greg-inspiron5767-* --keep-within 3d /mnt/TRUENAS/manjaro-home/borg
fish: No matches for wildcard 'sh:greg-inspiron5767-*'. See help wildcards-globbing.
borg prune --dry-run --list --info --log-json --keep-hourly 0 --keep-daily 10 --keep-weekly 0 --keep-monthly 6 --keep-yearly 0 -a sh:greg-inspiron5767-* --keep-within 3d /mnt/TRUENAS/manjaro-home/borg
^~~~~~~~~~~~~~~~~~~~~^
124
```

If i remove the "sh:" (in bash or fish) then it works.

```
borg prune --dry-run -v --list --info --keep-hourly 0 --keep-daily 10 --keep-weekly 0 --keep-monthly 6 --keep-yearly 0 -a greg-inspiron5767-* --keep-within 3d /mnt/TRUENAS/manjaro-home/borg
Keeping archive (rule: within #1):       greg-inspiron5767-2023-01-31-203529  Tue, 2023-01-31 20:35:30 [051208f999dcfabcc9b43b50b91b6e92622e013d15fc227afba43a3be5bd84f9]
Keeping archive (rule: within #2):       greg-inspiron5767-2023-01-31-192012  Tue, 2023-01-31 19:20:12 [191a904d25daa779f96a9b11e99b7305434a803f44120ae0b768829c55a550de]
Keeping archive (rule: within #3):       greg-inspiron5767-2023-01-31-191023  Tue, 2023-01-31 19:10:23 [7fcadf6abde5dcb04ff80e3cdf38223f7a0785c5846c71b1c7ade853e3b666a3]

etc etc ...
```

I assume i can just remove the prune prefix "{hostname}-" from the vorta setting for prune (i have not tried yet) but i thought i would ask about it.

**Comment by m3nu** [comment]
Duplicate of #1564, resolved in #1565

**Comment by JackDinn** [comment]
> Duplicate of #1564, resolved in #1565

Ah cool, I did search "prune" was just an odd title about borg version being incorrect so i passed over that post :( 

I have not touched borg or vorta for years, they work lovely. I have just built and installed a truenas system and because i just happened to be moving my backup over to that i assumed it had to be me that had messed up ^^ 

K, thx.

**Comment by m3nu** [comment]
Yeah, titles aren't always great. Thanks for checking in anyways! Fix will be out in v0.8.11
